http/http_errors.ts

```diff
@@ -98,6 +98,7 @@ export type ErrorStatusKeys = keyof typeof ERROR_STATUS_MAP;
 
 export interface HttpErrorOptions extends ErrorOptions {
   expose?: boolean;
+  headers?: Headers;
```

Comment:
```suggestion
  headers?: HeadersInit;
```

This will make it more usable.